packages/fxa-admin-panel/src/components/AccountSearch/index.tsx

```diff
@@ -49,20 +49,82 @@ export const GET_ACCOUNT_BY_EMAIL = gql`
   }
 `;
 
+export const GET_EMAILS = gql`
+  query fetchAll($search: String!) {
+    getEmailsLike(search: $search) {
+      email
+    }
+  }
+`;
+
 export const AccountSearch = () => {
   const [inputValue, setInputValue] = useState<string>('');
   const [showResult, setShowResult] = useState<Boolean>(false);
+  const [suggestions, setSuggestion] = useState<Array<string>>([]);
+  const [text, setText] = useState<string>('');
   const [getAccount, { loading, error, data, refetch }] = useLazyQuery(
     GET_ACCOUNT_BY_EMAIL
   );
+  const [getEmailLike, { data: data1, refetch: refetch1 }] = useLazyQuery(
```

Comment:
Let's find better names than `data1` and `refetch1`, something with `email` in it maybe